dev/com.ibm.ws.session.cache/resources/com/ibm/ws/session/store/cache/resources/WASSessionCache.nlsprops

```diff
@@ -19,4 +19,44 @@
+PROP_HIT_ERROR.explanation=An operation against the cache failed due to the specified exception.
+PROP_HIT_ERROR.useraction=See the exception for more details. The JCache provider's configuration might need to be updated.
+
+STORE_SESS_ERROR=SESN0303E: An exception occurred when storing a session in the cache. The exception is: {0}.
+STORE_SESS_ERROR.explanation=An operation against the cache failed due to the specified exception.
+STORE_SESS_ERROR.useraction=See the exception for more details. The JCache provider's configuration might need to be updated.
+
+LOAD_VALUE_ERROR=SESN0304E: An exception occurred when reading in an object of the application data for a session from the cache. The exception is: {0}.
+LOAD_VALUE_ERROR.explanation=An operation against the cache failed due to the specified exception.
+LOAD_VALUE_ERROR.useraction=See the exception for more details. The JCache provider's configuration might need to be updated.
+
+ERROR_CACHE_ACCESS=SESN0305E: An exception occurred while attempting to access the cache. The exception is: {0}.
+ERROR_CACHE_ACCESS.explanation=An operation against the cache failed due to the specified exception.
+ERROR_CACHE_ACCESS.useraction=See the exception for more details. The JCache provider's configuration might need to be updated.
+
+ERROR_SESSION_INVAL=SESN0306E: An exception occurred when invalidating a session in the cache. The exception is: {0}.
+ERROR_SESSION_INVAL.explanation=An operation against the cache failed due to the specified exception.
+ERROR_SESSION_INVAL.useraction=See the exception for more details. The JCache provider's configuration might need to be updated.
+
+ERROR_SESSION_INIT=SESN0307E: An exception occurred when initializing the cache. The exception is: {0}.
+ERROR_SESSION_INIT.explanation=An operation against the cache failed due to the specified exception.
+ERROR_SESSION_INIT.useraction=See the exception for more details. The JCache provider's configuration might need to be updated.
+
+WARN_CONFIG_INVALID_HTTPSESSIONCACHE=SESN0308W: Invalid or missing httpSessionCache configuration detected. {0}
+WARN_CONFIG_INVALID_HTTPSESSIONCACHE.explanation=The sessionCache feature was unable to fully initialize due to a missing or invalid configuration.
+WARN_CONFIG_INVALID_HTTPSESSIONCACHE.useraction=Verify the server configuration defines a httpSessionCache that includes a libraryRef attribute.
+
+ERROR_CONFIG_EMPTY_LIBRARY=SESN0309E: The session cache library {0} is empty. {1}
+ERROR_CONFIG_EMPTY_LIBRARY.explanation=The sessionCache feature could not find a JCache provider to load.
+ERROR_CONFIG_EMPTY_LIBRARY.useraction=Verify the library contains a JCache provider, and all paths are specified correctly.
+
+INTERNAL_SERVER_ERROR=Internal Server Error
+
+SESSION_CACHE_CONFIG_MESSAGE=Verify the server configuration contains a valid httpSessionCache configuration. See the sample configuration below:{0}
```

Comment:
Add a "that:"
Verify **that** the server configuration contains a valid httpSessionCache configuration.

We can't use directional words such as "below" or "above." These words are not very helpful for visually impaired users who get help from screen readers. How about simply omitting the word "below" and using a period instead of a colon, like so?
"See the sample configuration. {0}"